core-bundle/src/Resources/contao/library/Contao/Image.php

```diff
@@ -706,7 +706,8 @@ public static function getHtml($src, $alt='', $attributes='')
 			}
 		}
 
-		$objFile = new File($src);
+		// We know it's a file here so we can disable the directory check for performance reasons
+		$objFile = new File($src, false);
```

Comment:
Shouldn’t the `is_dir()` check be cached by PHP anyways because of the `is_file()` call on line 687?
To be sure that `$src` is a directory here would have to check `is_file()` in line 699 too.

Reply:
Yep but the realpath cache only works on systems where `open_basedir` isn‘t set, so it‘s maybe still worth optimizing.